src/identity/components/GlobalHeader/GlobalHeaderDropdown/index.tsx

```diff
@@ -172,27 +171,30 @@ export class GlobalHeaderDropdown extends React.Component<Props, State> {
       >
         {/* Multi-org UI tickets #4051 and #4047, when user only has 1 account or 1 org, switch button is disabled */}
         {!onlyRenderSubmenu && typeAheadMenuOptions.length > 1 && (
-          <Dropdown.Item
-            onClick={this.toggleShowTypeAheadMenu}
-            className="global-header--switch-button"
-          >
-            <FlexBox
-              justifyContent={JustifyContent.SpaceBetween}
-              alignItems={AlignItems.Center}
+          <>
+            <Dropdown.Item
+              onClick={this.toggleShowTypeAheadMenu}
+              className="global-header--switch-button"
             >
-              <span className="global-header--align-center">
-                {iconEl}
-                {textEl}
-              </span>
-              {showTypeAheadMenu === false && (
-                <Icon
-                  glyph={IconFont.CaretRight_New}
-                  className="button-icon"
-                  style={{marginRight: 0}}
-                />
-              )}
-            </FlexBox>
-          </Dropdown.Item>
+              <FlexBox
+                justifyContent={JustifyContent.SpaceBetween}
+                alignItems={AlignItems.Center}
+              >
+                <span className="global-header--align-center">
+                  {iconEl}
+                  {textEl}
+                </span>
+                {showTypeAheadMenu === false && (
+                  <Icon
+                    glyph={IconFont.CaretRight_New}
+                    className="button-icon"
+                    style={{marginRight: 0}}
+                  />
+                )}
+              </FlexBox>
+            </Dropdown.Item>
+            <hr className="line-break" />
```

Comment:
make it part of the switch button, so it only renders when switch button is supposed to.

Reply:
Looks good - nice fix! This is def cleaner than what I was proposing.